pytest-selenium/tests/test_attribution.py

```diff
@@ -0,0 +1,23 @@
+from pages.content import Content
+from tests import markers
+
+
+@markers.test_case("C476302")
```

Comment:
this doesn't match this test

Reply:
do you mean this doesn't match the test case in test rail?

Reply:
yes, the test in testrail is testing something different

Reply:
I think we need to talk to understand this conflict

Reply:
I finally figured out what Tom is saying and yes my test is not checking the intended functionality. Thanks for pointing it out Tom. 
I actually need to check the urls that are within the citation section (section url) refer to the current page url

Reply:
@TomWoodward I am rewriting this script.
Section URL within the attribution always defaults to current page.
Do we also know that the Book url & Access for free url always points to Chapter 1 or will there be a change to it?

Reply:
they point to the "Default page" of the book, which is different for different books but shouldn't change. its defined as the first page in the first chapter (recursively) so if there are units its the first page of the first chapter of the first unit. 

this is to avoid defaulting to the preface, which we generally try to ignore

Reply:
i'm assuming that there will be some redirects to rex from openstax.org in the future. I'm seeing openstax.org in these links but they return 404 when clicked. 
![image](https://user-images.githubusercontent.com/8730430/58033060-68755780-7ae9-11e9-8313-692373e7502b.png) cc: @TomWoodward

Reply:
Based on Tom's comments this test would just cover the section URL and I am writing separate tests for the Book URL and access for free URL

Reply:
@m1yag1 the openstax.org links are correct, that is what the rex url will be after launch

Reply:
👍 . it would be nice to continue to use the `base_url` we pass in to build the url to do the verification ... we can make it work.

Reply:
@m1yag1 
Since the section url in attribution is pointing to production url, I did not find an easy way other than hardcoding the base url. But if its achievable the other way around, we can do that.